Practice/kruglov/home_task_4/PyLec4T3-4.py

```diff
@@ -0,0 +1,34 @@
+# 3
+d = [0, 3, 24, 2, 3, 7]
+b = []
+
+i = 0
+s = len(d)
+while i < s:
+    c = min(d)
+    b.append(c)
+    d.pop(d.index(c))
+    i += 1
+
+print(b)
+
+# 3.1
```

Comment:
Almost meets the task) But it's expected that u-element will take the place of min element as well.

Reply:
Fixed